README.md

```diff
@@ -0,0 +1,164 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Bookstore add Reducer and Action</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Math Magicians <a name="about-project"></a>
+
+**Math magicians** Bookstore is a simple website that displays a list of books and allows you to:
```

Comment:
- [ ] Let's try to be thorough in our documentation. This is a `BookStore` project, but your README contains information describing the ` `Math magicians``. I'd suggest you fix the title of your project. ✍🏻